app/forms/flood_risk_engine/steps/local_authority_address_form.rb

```diff
@@ -15,28 +15,10 @@ def self.params_key
         :local_authority_address
       end
 
-      property :postcode, virtual: true
-
-      # read only param for displaying the Postcode in the view
-      def postcode
-        return enrollment.address_search.postcode if enrollment.address_search.present?
-
-        return enrollment.organisation.primary_address.postcode if enrollment.organisation.primary_address.present?
-
-        ""
-      end
-
       private
 
       def initialize(model, enrollment)
         super(model, enrollment)
-
-        # Consensus is that the original drop down list should be displayed when a user clicks Back,
-        # If address fields required local_authority_address_back can include partials 'shared/address_fields'
-        # or 'shared/display_address'
-        partial = "flood_risk_engine/enrollments/steps/local_authority_address_back"
-
-        @partial_to_render = partial if enrollment.organisation.primary_address.present?
       end
```

Comment:
Yes agreed, I don't think requirements will change now so no need to potentially render anything different to original view

To complete I will definitely rationalise address forms and partials to use new base classes